tests/infer/test_inference.py

```diff
@@ -243,7 +256,12 @@ def guide(alpha0, beta0, alpha_n, beta_n):
         pyro.sample("lambda_latent", gamma_dist(alpha_q, beta_q))
 
     adam = optim.Adam({"lr": .0003, "betas": (0.97, 0.999)})
-    elbo = elbo_impl(strict_enumeration_warning=False)
+    if elbo_impl is RenyiELBO:
+        elbo = elbo_impl(alpha=0.2, num_particles=3, strict_enumeration_warning=False)
+        if gamma_dist is ShapeAugmentedGamma:
+            pytest.skip("RenyiELBO currently does not work with Rejector!")
```

Comment:
I made the issue #1222 for this skip.

Reply:
itd be great if you could link the issue in the comment

Reply:
I'll add it after @neerajprad 's review.